code/modules/clothing/spacesuits/chronosuit.dm

```diff
@@ -153,7 +152,7 @@
 
 /obj/item/clothing/suit/space/chronos/proc/phase_3(mob/living/carbon/human/user, turf/to_turf, phase_in_ds)
 	if(teleporting && activated && user)
-		user.forceMove(to_turf)
+		do_teleport(user, to_turf, no_effects = TRUE, channel = TELEPORT_CHANNEL_BLINK)
```

Comment:
bad changes, this breaks it (fairly sure, since I changed it BACK to forceMove) and also it's an admin item and should be immune

Reply:
lore-wise it's literally phasing in and out of time, not "teleporting"

Reply:
Nothing that teleports should use forceMove to perform the teleportation. This won't work on centcom or anywhere without teleportation however

Reply:
It needs to work on centcom because it's an admin item and if the teleport fails it breaks the shit out of the suit